common-files/utils/gas.mjs

```diff
@@ -0,0 +1,69 @@
+/* ignore unused exports */
+
+import axios from 'axios';
+import logger from './logger.mjs';
+
+/**
+ * Estimate the amount of gas that will be needed to submit a transaction (tx)
+ *
+ * The underlying RPC `eth_estimateGas` will check the balance of the sender,
+ * this means that even though the call doesn't consume any gas,
+ * the `from` address must have enough gas to execute the tx
+ *
+ * @async
+ * @function estimateGas
+ * @param {object} tx A tx object
+ * @param {Web3} web3
+ * @param {number} gasDefault Has default
+ * @param {number} gasMultiplier Buffer to apply to estimated gas - Has default
+ * @returns {Promise<number>}
+ */
+export async function estimateGas(tx, web3, gasDefault = 4000000, gasMultiplier = 2) {
```

Comment:
can you check `common-files/utils/web3.mjs`? there is an internal function called `estimateGas`. Would you mind deleting that and using yours instead?

Reply:
#1286